e2e/remix-e2e/tests/nx-remix.spec.ts

```diff
@@ -52,7 +52,9 @@ describe('remix e2e', () => {
   }, 120000);
 
   describe('--directory', () => {
-    it('should create src in the specified directory', async () => {
+    // todo - Application does not have a directory option
```

Comment:
Can we change this to `TODO:` or `TODO(Coly010):` so we can search for it later?

Reply:
Yep, sure thing!